presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -200,6 +200,11 @@ Specifier Description
 
 .. warning:: The following specifiers are not currently supported: ``%D %U %u %V %X``
 
+.. warning:: Timestamp is truncated to milliseconds
+
+.. note:: ``Fraction of second`` behaves the same way as ``Microseconds`` from the MySQL reference manual (for the ``date_format`` function)
+    Similarly to MySQL, it accepts between 6 and 9 digits of input.
```

Comment:
I didn't know that MySQL actually allows 1-9 digits when parsing "%f". I trusted MySQL's documentation. Given this, I believe maybe the explanation of equivalency is no longer needed. We are doing the exact same thing. (I thought we wasn't.) We are just giving it different names.

But, please change the documentation in Presto to reflect the accurate meaning. Thank you!